doc/api/deprecations.md

```diff
@@ -850,6 +861,7 @@ or `module.exports` instead.
 [`Server.getConnections()`]: net.html#net_server_getconnections_callback
 [`Server.listen({fd: <number>})`]: net.html#net_server_listen_handle_backlog_callback
 [`SlowBuffer`]: buffer.html#buffer_class_slowbuffer
+[`asyncResource.makeCallback()`]: async_hooks.md#asyncresource_makecallback
```

Comment:
It seems this should be
`async_hooks.html#async_hooks_asyncresource_makecallback_fn_thisarg_args`

Reply:
Done.